wasm/js/renderer.js

```diff
@@ -874,4 +874,10 @@ Rive.onRuntimeInitialized = function () {
   );
   Rive["disableFPSCounter"] = _animationCallbackHandler.disableFPSCounter;
   _animationCallbackHandler.onAfterCallbacks = flushCanvasRenderers;
+
+  Rive["cleanup"] = function () {
+    if (_rectanizer) {
+      _rectanizer.delete();
```

Comment:
Was this a leak?

Reply:
No but if you run the leak test in the example app it will report leaks if you don't call this (we allocate it once like a singleton/manager and it detects that as leaked memory).